mvds/README.md

```diff
@@ -104,7 +104,7 @@ Nodes MAY have two modes with which they can send records: `BATCH` and `INTERACT
  - All records that require retransmission are added to the payload, given `Send Epoch` has been reached.
 
 <p align="center">
-    <img src="./assets/mvds/interactive.png" />
+    <img src="/assets/mvds/interactive.png" />
```

Comment:
the dot is there on purpose, is there a good reason not to use relative path here?

Reply:
I'd probably not touch this file in this PR

Reply:
@oskarth its cause I moved all MVDS related specs to an MVDS path, I think that makes sense.